daemon/cmd/daemon_main.go

```diff
@@ -235,6 +235,9 @@ func initializeFlags() {
 	flags.Bool(option.EnableEndpointRoutes, defaults.EnableEndpointRoutes, "Use per endpoint routes instead of routing via cilium_host")
 	option.BindEnv(Vp, option.EnableEndpointRoutes)
 
+	flags.Bool(option.EnableBPFFibCustomRules, defaults.EnableBPFFibCustomRules, "Use custom fib rule lookup for BPF Host Routing multi-device deployments")
+	option.BindEnv(Vp, option.EnableBPFFibCustomRules)
+
```

Comment:
Nit: following [Go naming conventions](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) for acronyms it should be

```suggestion
	flags.Bool(option.EnableBPFFIBCustomRules, defaults.EnableBPFFIBCustomRules, "Use custom fib rule lookup for BPF Host Routing multi-device deployments")
	option.BindEnv(Vp, option.EnableBPFFIBCustomRules)

```

Reply:
Thank you. I will modify the code according to good coding suggestions